service/src/intTest/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+server:
```

Comment:
Consider adding an integration test for healthcheck? That way there is at least something in this intTest src dir.